test/stress.c

```diff
@@ -92,170 +92,170 @@ int main(int argc, char *argv[])
     TestKeyEntry *key_entry;
 
     for (i = 1; i < argc; i++) {
-	if (strcmp(argv[i], "-init") == 0)
-	    flag_random_init = 1;
-	else if (strcmp(argv[i], "-extra") == 0)
-	    flag_random_extra = 1;
-	else if (strcmp(argv[i], "-verbose") == 0) {
-	    flag_verbose = 1;
+        if (strcmp(argv[i], "-init") == 0)
+            flag_random_init = 1;
+        else if (strcmp(argv[i], "-extra") == 0)
+            flag_random_extra = 1;
+        else if (strcmp(argv[i], "-verbose") == 0) {
+            flag_verbose = 1;
             logger = verbose_logger;
         } else if (strcmp(argv[i], "-loop") == 0 && argv[i + 1])
-	    flag_loop = atoi(argv[++i]);
-	else if (strcmp(argv[i], "-stdin") == 0) {
+            flag_loop = atoi(argv[++i]);
+        else if (strcmp(argv[i], "-stdin") == 0) {
             input_fd = 0;
-	    get_input = &read_from_fd;
+            get_input = &read_from_fd;
         } else if (strcmp(argv[i], "-file") == 0 && argv[i + 1]) {
             input_fd = open(argv[i + 1], O_RDONLY);
             if (input_fd < 0) {
                 fprintf(stderr, "failed to open '%s'\n", argv[i + 1]);
                 exit(1);
             }
-	    get_input = &read_from_fd;
+            get_input = &read_from_fd;
             i++;
         } else {
-	    printf("Usage: %s [-init] [-extra] [-loop N] [-stdin]\n", argv[0]);
-	    printf("\t-init           Random initial configuration\n");
-	    printf("\t-extra          Random change all configurations during input.\n");
-	    printf("\t                This is usually unexpected.\n");
-	    printf("\t-stdin          Get random input from stdin\n");
-	    printf("\t-loop N         How many iterations to test (default infinite=-1)\n");
+            printf("Usage: %s [-init] [-extra] [-loop N] [-stdin]\n", argv[0]);
+            printf("\t-init           Random initial configuration\n");
+            printf("\t-extra          Random change all configurations during input.\n");
+            printf("\t                This is usually unexpected.\n");
+            printf("\t-stdin          Get random input from stdin\n");
+            printf("\t-loop N         How many iterations to test (default infinite=-1)\n");
             printf("\t-verbose        Verbose\n");
-	    exit(1);
-	}
+            exit(1);
+        }
     }
 
     /* Initialize for testing */
     for (key_entry = chewing_test_special_keys; key_entry->key; key_entry++)
-	num_special_key++;
+        num_special_key++;
 
     /* Initialize libchewing */
     chewing_sys_path = getenv("CHEWING_PATH");
     if (!chewing_sys_path)
-	chewing_sys_path = CHEWING_DATA_PREFIX;
+        chewing_sys_path = CHEWING_DATA_PREFIX;
 
     /* for the sake of testing, we should not change existing hash data */
     userphrase_path = get_test_userphrase_path();
 
     for (i = 0; i != flag_loop; i++) {
-	ChewingContext *ctx;
-	clean_userphrase();
-	ctx = chewing_new2(chewing_sys_path, userphrase_path, logger, NULL);
+    ChewingContext *ctx;
```

Comment:
ditto. This looks weird, too.

BTW, I didn't check out your change. I just look on github web ui. Maybe your change is good. I'm not sure.